CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 All notable changes to Explorer for Endevor will be documented in this file.
 
 ## [Unreleased]
+
+## [0.9.1] 2020-02-28
+# Changed
+- Version 0.9.1 Released
```

Comment:
Also, we dont need to mention that we have released that version under every release (e.g. - Version 0.9.1 Released) - I would say that's obvious from the headline already - [0.9.1] 2020-02-28 

If we could remove those lines in changelog in all the previous releases, it would be nice :)